README.md

```diff
@@ -1,2 +1,39 @@
 # embedded-common
 Embedded Driver common files
+
+## Clone this repository
+If you really need this repository you can check it out with the following command:
```

Comment:
Cloning instruction is not needed here, I'd add the following note instead (also remove the subsection header "Clone this repository").

> This repository contains the Sensirion i2c abstraction layer and commonly shared code. It is a dependency of the Sensirion embedded drivers and it should not be necessary to clone it independently.

Then follow up with the last paragraph "We recommend to download ..."

Reply:
Done